expression/builtin_time_vec.go

```diff
@@ -136,11 +136,28 @@ func (b *builtinFromUnixTime2ArgSig) vecEvalString(input *chunk.Chunk, result *c
 }
 
 func (b *builtinSysDateWithoutFspSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinSysDateWithoutFspSig) vecEvalTime(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	loc := b.ctx.GetSessionVars().Location()
+	nowTs, err := getStmtTimestamp(b.ctx)
+	if err != nil {
+		return err
+	}
```

Comment:
It seems these lines are not used. Can we delete them?

Reply:
`nowTs` is used later. To get a `now` value.